tests/test-definition.yaml

```diff
@@ -13,11 +13,15 @@ dimensions:
   - name: superset-latest
     values:
       - 4.1.1
+      # - 4.1.1,oci.stackable.tech/razvan/superset:4.1.1-stackable0.0.0-dev
   - name: ldap-authentication
     values:
       - no-tls
       - insecure-tls
       - server-verification-tls
+  - name: opa
```

Comment:
I would prefer to rename it to `opa-latest`, as we are only testing against a single version for runtime reasons

Reply:
Yeah, I [renamed](https://github.com/stackabletech/superset-operator/pull/582/commits/bcd774f60c865b535f7e8088169e16a585439bf5) the dimension and not the test. We may need to support multiple opa versions sometime.